scripts/bootstrap.sh.in

```diff
@@ -79,8 +83,10 @@ install_kubelet() {
 }
 
 run_kubeadm_init() {
-        ${SALT_CALL} --local --retcode-passthrough state.apply metalk8s.kubeadm.init saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
```

Comment:
Why?

Reply:
Because we have several needs:
1- We install `python-netaddr` used when rendering `apiserver.sls` state which is installed in preflight (Will be fixed, as we not longer compute the ip and just fix it in the pillar)
2- We need to restart salt-minion in `ca.sls` state to give access to the `signing_policies` so we need to call `ca.sls` and `apiserver.sls` with 2 different salt-call

Reply:
Right. So we'll need an `orchestrate` thing at some point, instead of calling things 'in the right order' like we do now.

Reply:
Yes we will need some orchestrator